ledger-core/network/servicenetwork/servicenetwork.go

```diff
@@ -45,7 +46,7 @@ type ServiceNetwork struct {
 
 	// subcomponents
 	RPC                controller.RPCController   `inject:"subcomponent"`
-	PulseAccessor      storage.PulseAccessor      `inject:"subcomponent"`
+	PulseAccessor      appctl.PulseAccessor       `inject:"subcomponent"`
```

Comment:
I'm not sure that it's good for network compartment that it haas appctl inside. I thought that they shouldn't be coupled. But I might be wrong.
I think it would be better to have here interface something like 
```
type Pulser interface {
   GetLatestPulse(context.Context) (pu, error)
}
```